build/ci/job-template.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       nightlyBuildRunPath: $(Build.SourcesDirectory)/bin/AnyCPU.$(_configuration)/Microsoft.ML.NightlyBuild.Tests/$(_targetFramework)
       packageUpdaterProjPath: $(Build.SourcesDirectory)/test/Microsoft.ML.NugetPackageVersionUpdater/Microsoft.ML.NugetPackageVersionUpdater.csproj
       versionFilePath: $(Build.SourcesDirectory)/test/Microsoft.ML.NugetPackageVersionUpdater/latest_versions.txt
-      PROCDUMP_PATH: '$(Build.SourcesDirectory)/Tools/procdump/'
+      PROCDUMP_PATH: '$(Build.SourcesDirectory)/Procdump_Files/'
```

Comment:
This will create a separate directory under the repo root just for Procdump. 
What is the rationale for not keeping it under Tools?

Reply:
Good point, thank you for the heads up.